integration-tests/ui/src/test/resources/migration/MIGR_TOOLS/xsk-test-app/xsk-test-app-copy.hdiconfig

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Why the filename is `xsk-test-app-copy.hdiconfig`? it seems that in the `*.hdi` file it's referenced as `xsk-test-app.hdi`.

Reply:
copy/paste omission